script.js

```diff
@@ -1,4 +1,72 @@
+
+// Project 1: scissors paper stone 
+// user input scissors or paper, or stone
+// input validation for "scissors" "paper" "stone"
+// assign 0,1,2 to scissors, paper, stone
+// random generate number 0,1,2
+// check input vs comPlay
+// winCondition: scissors > paper || paper > stone || stone > scissors 
+// drawCondition: input == computerRandom
+// updated for formatting 
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  input = input.toLowerCase();
+  var myOutputValue = "";
+  var input1 = "";
+
+//input validation
+if (input != "scissors" || 
+input != "paper" ||
+input != "stone") { 
+  myOutputValue = "Please try again, type 'scissors' 'paper' or 'stone'"
   return myOutputValue;
+}  
+  var comPlay = randomNumber()
+console.log (comPlay);
+
+// scissors = 0
+// paper = 1
+// stone = 2 
+
+if(comPlay == 0) {
+  input1 = "scissors"
+}
+
+if(comPlay == 1) {
+  input1 = "paper"
+}
+
+if(comPlay == 2) {
+  input1 = "stone"
+}
+
+// check user vs. computer outomce
+if (input == "scissors" && comPlay == 1||
+input == "paper" && comPlay == 2 ||
+input == "stone" && comPlay == 0) {
+var outcome = "you win"
+}
+
+if (input == "scissors" && comPlay == 2||
+input == "paper" && comPlay == 0 ||
+input == "stone" && comPlay == 1) {
+outcome = "you lose"
+}
```

Comment:
You can consider adding a comment above each code block at lines 44, 50, 56 - explaining which condition each block is checking for. For e.g. // Checking for user win // Checking for draw  etc.
Even though it seems counter-intuitive since the outcome is already written a few lines down, sometimes people will want to skim through your code quickly and leaving comments on code blocks like functions, if statements and loops will make it easier and clearer for everyone.